packages/browser-repl/config/webpack.config.base.js

```diff
@@ -1,35 +1,53 @@
-const path = require('path');
+const webpack = require('webpack');
 
 module.exports = {
+  target: 'web',
   stats: 'errors-only',
   resolve: {
     extensions: ['.tsx', '.ts', '.jsx', '.js', '.less'],
-    alias: {
-      // imports in service-provider-core that can break the browser build
-      'whatwg-url': path.resolve(__dirname, 'plain-url.js'),
+    fallback: {
+      // node specific and don't require a polyfill
+      zlib: false,
+      v8: false,
+      fs: false,
+      crypto: false,
+      // node specific and require a polyfill
+      path: require.resolve('path-browserify'),
+      stream: require.resolve('stream-browserify'),
+      buffer: require.resolve('buffer/'),
+      util: require.resolve('util/'),
```

Comment:
Maybe also leave a comment about these versions? I admittedly don’t quite understand how this is being used :)

Reply:
I added a list of dependencies that use these built-ins and so require a polyfill to be able to run the code in the browser in https://github.com/mongodb-js/mongosh/pull/1355/commits/aece959fced5376e18a4389fcf0841abb35f48b3. Is this something that clarifies it, or were you looking for something else? I am happy to clarify more, just not sure what exactly 😊

Reply:
No, all good – I didn't realizd that adding the slash will make the Node.js resolution algorithm use the polyfill and that we actually had packages installed from npm that had these literal names :)

Reply:
Ahhhh, gotcha! Yeah, I misunderstood then and will add a comment about this too, they are indeed confusingly published under the same name and adding a slash helps to resolve them "correctly"